### PR TITLE
Add Medium Speed Contest rules

### PR DESCRIPTION
**Pull request by VictorDenisov**

### CODE REVIEW

**Comment by zcsahok** [comment]
I believe it works even with plain TLF configs. At least with these ones it passed the test:

```
ONE_POINT
UNIQUE_CALL_MULTI=BAND
```

File: rules/mst/mst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# Provided by: Victor Denisov N6DVS
+# Last Verified: 2023-06-12
+########################
+#
+CONTEST_MODE
+LOGFILE=mst.log
+CABRILLO=UNIVERSAL
+ONE_POINT
+UNIQUE_CALL_MULTI=ALL
+#SERIAL_OR_SECTION
```
Comment (zcsahok) [inline rules/mst/mst:14]: Unused lines could be dropped. (the ones commented out + PLUGIN_CONFIG)

File: rules/mst/mst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#ALT_5=
+#ALT_6=
+#ALT_7=
+#ALT_8=
+#ALT_9=
+#
+#SEND_DE
+#
+CABRILLO-QSO-FORMAT=SST
+CABRILLO-EXCHANGE=vic ca
```
Comment (zcsahok) [inline rules/mst/mst:67]: Not sure if this is relevant/correct. No Cabrillo has to be submitted.

File: test/rules/mst/rules/mst.py [hunk 3]
```diff
@@ -0,0 +1 @@
+../../../../rules/mst/mst.py
```
Comment (zcsahok) [inline test/rules/mst/rules/mst.py:1]: pls remove the symlink to `mst.py`

**Review by zcsahok — CHANGES_REQUESTED** [review]
Just added some comments.

**Comment by VictorDenisov** [comment]
Done

**Review by zcsahok — APPROVED** [review]

**Pull request merged**